### ISSUE TITLE
[feat]: /docs/components/magic-card

### ISSUE THREAD

**Issue by Shreyas-29** [magicuidesign/magicui#277]
The Magic Card present till date on MagicUI is cool.

**Describe the solution you'd like**
But,
If there are multiple cards in a bento grid layout where I can add more subtle hover spotlight effect both for the inner card and border.

**Describe alternatives you've considered**
Here are some alternatives:

- https://lunarui.dev/components/react/feature-blocks/spotlight-card (PAID)
- https://cruip.com/how-to-create-a-spotlight-card-hover-effect-with-tailwind-css/#create-a-reusable-spotlight-component-for-nextjs (Animations not working properly)

**Additional context**
Basically I want to create an effect like these websites:

- https://wope.com/content-assistant/
- https://linear.app/features

<img width="1029" alt="Screenshot 2024-08-22 at 4 56 31 PM" src="https://github.com/user-attachments/assets/c2230603-ce95-482e-b8f9-10cb2bb415d6">

**Comment by itsarghyadas** [comment]
@Shreyas-29 We’ve already added this border effect in PR https://github.com/magicuidesign/magicui/pull/467 so I’m closing this PR in favor of that one.
